src/constants/responses.ts

```diff
@@ -27,3 +27,5 @@ export const ROLE_ADDED = "Role added successfully";
 export const NAME_CHANGED = "User nickname changed successfully";
 
 export const ROLE_REMOVED = "Role Removed successfully";
+export const VERIFICATION_STRING =
+  "Like to verify yourself? click the above link and authorize real dev squad to manage your discord data";
```

Comment:
The below message sounds clearer and more direct to the user:

"Please verify your identity by clicking the link above and granting authorization to the Real Dev Squad. This will allow us to manage your Discord data."